docs/changelog-fragments.d/957.internal.md

```diff
@@ -0,0 +1,9 @@
+Added new subcommand *settings* to output all current ansible-navigator
+settings to the menu or to stdout.
+
+Here is an example invocation:
+```bash
+ansible-navigator settings
+```
+
+-- by {user}`alisonlhart` {user}`thedoubl3j`
```

Comment:
I think it'd probably be nicer to have an "and" here:
```suggestion
-- by {user}`alisonlhart` and {user}`thedoubl3j`
```

Reply:
Agreed. I'm adding it in.